spec/marketing_only_app/translations/en.json

```diff
@@ -0,0 +1,21 @@
+{
+  "app": {
+    "name": {
+      "title": "Name for App: Marketing Only App",
+      "value": "Marketing Only App"
+    },
+    "description": {
+      "title": "Description for App: Marketing Only App",
+      "value": "Bring your Zendesk into your other service"
+    },
+    "long_description": {
+      "title": "Long Description for App: Marketing Only App",
+      "value": "Zendesk for Marketing Only App allows you to integrate your Zendesk Support helpdesk with your other service."
```

Comment:
@chucknado sorry for confusion, this file (along with any other in the `spec/` directory) is used exclusively in tests. The contents are probably not important.